app/Models/Forum/PollOption.php

```diff
@@ -36,6 +36,7 @@ public function post()
     {
         return $this
             ->belongsTo(Post::class, 'topic_id', 'topic_id')
+            ->withTrashed()
```

Comment:
having some associations include deleted records and others not without any labeling seems confusing